account_payment_batch_process/README.rst

```diff
@@ -0,0 +1,36 @@
+
+=====================================
+Batch Payments Processing for Odoo 11
+=====================================
+
+This module implements Process Payments in Batch for Odoo 11.
+
+
+Usage
+=====
+
+#.  Go to Invoicing > Sales > Customer Invoices or 
+    Invoicing > Purchases > Vendor Bills, and select invoices to pay and
+    press Action > Batch Payments.
+
+#.  It will show a window with information on the balance of each 
+    selected invoice. Select journal, payment date,  payment amount 
+    total and add amount to pay on each line of invoice.
+
+#.  Next, press button "Make Payments" to create payments in batch.
+
+
+Maintainer
+----------
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit https://odoo-community.org.
```

Comment:
@faryrosa Can you use the new readme structure from:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst